tutorials/locations/azure-example-location.bom

```diff
@@ -19,7 +19,6 @@ brooklyn.catalog:
       regionId: North Europe
       hardwareId: BASIC_A2
       loginUser: user
-      installDevUrandom: true
```

Comment:
This is the default anyway - see https://github.com/apache/brooklyn-server/blob/0.10.0/locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocationConfig.java#L190-L191